content/en/_redirects

```diff
@@ -13,36 +13,38 @@
 /blog/announcing_kubeflow_0.2/         https://medium.com/kubeflow/kubeflow-0-2-offers-new-components-and-simplified-setup-735e4c56988d
 
 /docs/guides/components/components     /docs/components/
-/docs/guides/pipelines/deploy-pipelines-service/  /docs/pipelines/pipelines-quickstart/
+/docs/guides/pipelines/deploy-pipelines-service/  /docs/components/pipelines/pipelines-quickstart/
 
 # Removed the "guides" section from the doc URLs.
 /docs/guides/*  /docs/:splat
 
 # Merged duplicate page pipelines-samples.md into build-pipeline.md
-/docs/pipelines/pipelines-samples/	   /docs/pipelines/build-pipeline/
+/docs/pipelines/pipelines-samples/	   /docs/components/pipelines/build-pipeline/
 
 # Removed redundant UI guide. Quickstart is a better destination.
-/docs/pipelines/pipelines-ui/	         /docs/pipelines/pipelines-quickstart/
+/docs/pipelines/pipelines-ui/	         /docs/components/pipelines/pipelines-quickstart/
 
 # Restructured the pipelines docs.
-/docs/pipelines/output-viewer/                      /docs/pipelines/sdk/output-viewer/
-/docs/pipelines/pipelines-metrics/                  /docs/pipelines/sdk/pipelines-metrics/
-/docs/pipelines/build-component/                    /docs/pipelines/sdk/build-component/
-/docs/pipelines/install-sdk/                        /docs/pipelines/sdk/install-sdk/
-/docs/pipelines/lightweight-python-components/      /docs/pipelines/sdk/python-function-components/
-/docs/pipelines/sdk/lightweight-python-components/  /docs/pipelines/sdk/python-function-components/
-/docs/pipelines/build-pipeline/                     /docs/pipelines/tutorials/build-pipeline/
-/docs/pipelines/pipelines-tutorial/                 /docs/pipelines/tutorials/cloud-tutorials/
-/docs/pipelines/tutorials/pipelines-tutorial/       /docs/pipelines/tutorials/cloud-tutorials/
-/docs/gke/pipelines-tutorial/                       /docs/pipelines/tutorials/cloud-tutorials/
-/docs/gke/pipelines/pipelines-tutorial/             /docs/pipelines/tutorials/cloud-tutorials/
+/docs/pipelines/*                                   /docs/components/pipelines/:splat
```

Comment:
Please move line 28 to the bottom of the file.

Redirects such as the following are not evaluated if they are caught by the catch-all redirect.

/docs/pipelines/install-sdk/   ->  /docs/components/pipelines/sdk/install-sdk/

Reply:
@joeliedtke thank you for the recommendation. Does moving line 28 to the end change anything in the automations? i.e. should I make it clear through a comment that this has to be the case in the future?

I see. Should we move ahead despite your second point? How should we proceed?

Reply:
To clarify, those aren't meant as separate points. The second item is an example of a redirect that is broken due to the position of the catch-all redirect. Here is a document that discusses the [redirect processing order](https://docs.netlify.com/routing/redirects/#rule-processing-order).

I would recommend adding any new catch-all redirects, such as line 28, to the bottom of the file to ensure that existing redirects are used. It may be a good idea to add a comment indicating that catch-all redirects must be added to the end of the file.

I'll make a note to review the file soon to see if there are any other catch-all redirects that should be moved.

Reply:
Oh! I didn't know this. That makes a lot of sense. Apologies for my lack of understanding so far.

I will, as you propose, move this catch-all to the bottom of the file under a comment of explanation, and add more alike.

I'm able to take on this task tomorrow.

Reply:
Done!